app/pages/Schema/SchemaConfig/List/SpaceStats/index.tsx

```diff
@@ -103,17 +103,23 @@ const SpaceStats = () => {
           onClick={handleSubmitStats}
           loading={loading || jobId !== null}
         >
-          {intl.get('schema.refresh')}
+          {intl.get(updateTime ? 'schema.refresh' : 'schema.startStat')}
         </Button>
-        <span className={styles.label}>{intl.get('schema.lastRefreshTime')}</span>
-        <span>
-          {updateTime ? dayjs(updateTime).format('YYYY-MM-DD HH:mm:ss') : null}
-        </span>
+        {(loading || jobId !== null || !updateTime) && <span className={styles.tip}>
+          {intl.get('schema.statTip')}
+        </span>}
+        {updateTime && jobId == null && !loading && <>
+          <span className={styles.label}>{intl.get('schema.lastRefreshTime')}</span>
+          <span>
+            {dayjs(updateTime).format('YYYY-MM-DD HH:mm:ss')}
+          </span>
+        </>}
```

Comment:
`updateTime && jobId == null && !loading` === `!(loading || jobId !== null || !updateTime)` ?

just make it
```ts
const showRefreshTime = updateTime && jobId == null && !loading;

showRefreshTime ? (...) : (...)
```